src/test/ui/parser/issues/issue-70583-block-is-empty-2.stderr

```diff
@@ -1,8 +1,10 @@
 error: unexpected closing delimiter: `}`
   --> $DIR/issue-70583-block-is-empty-2.rs:14:1
    |
+LL |         match self {
+   |                    - this delimiter might not be properly closed...
 LL |             ErrorHandled::Reported => {}}
-   |                                       -- block is empty, you might have not meant to close it
```

Comment:
Is any test left for the "block is empty" case? (This a somewhat common occurrence, due to helpful editors.)